src/System.ServiceProcess.ServiceController/src/System.ServiceProcess.ServiceController.csproj

```diff
@@ -92,8 +92,12 @@
     </Compile>
     <Compile Include="Microsoft\Win32\SafeHandles\SafeServiceHandle.cs" />
     <Compile Include="System\ServiceProcess\PowerBroadcastStatus.cs" />
-    <Compile Include="System\ServiceProcess\ServiceBase.cs" />
-    <Compile Include="System\ServiceProcess\ServiceController.cs" />
+    <Compile Include="System\ServiceProcess\ServiceBase.cs">
+      <SubType>Component</SubType>
```

Comment:
It is auto-generated by the VS. It means it extends the component type

Reply:
Can you please delete that? I don't think it is necessary for building and using VS.
We have it in bunch of *.csproj files, but I doubt it is consistently everywhere ... so it would be best to keep it intact in each project IMO.

Reply:
yeah I will already deleted them

Reply:
Yeah I have already deleted them :)